### ISSUE TITLE
No such file or directory (Alpine Linux)

### ISSUE THREAD

**Issue by myInstagramAlternative** [tiagorlampert/CHAOS#73]
Hello @tiagorlampert, wanna ask is there is any chance of updating Docker image? Current image use Alpine build of Go.

I get error when I try to use generated client on any Linux host except Alpine. I did some research and I've found that you can't use go builds on Linux if they were generated on Alpine.

I'd submit PR but Dockerfile have some errors and I'm not that familiar with containerized go builds.

So in short:
client don't work at all on classic linux host,
client works in Alpine container,
client works on Windows host (windows defender detects but that's not big deal)

Please try golang:latest

Thank you for your time.

**Comment by tiagorlampert** [comment]
Hello @myInstagramAlternative, thanks for your observation.

I've updated the docker image to golang:1.17.5 like you sugested, looks it work on other distros.
The only bad point is that the image size will increase, but i'ts ok.

The changes made are on this PR: [https://github.com/tiagorlampert/CHAOS/pull/74/files](https://github.com/tiagorlampert/CHAOS/pull/74/files)

Or can be tested using the image tiagorlampert/chaos:v5.0.2

Could you help me to test?

**Comment by myInstagramAlternative** [comment]
Hey @tiagorlampert!

Image tiagorlampert/chaos:v5.0.2 works now, linux client gets connected back to server.

Best regards,
